src/components/DropZone.tsx

```diff
@@ -1,9 +1,27 @@
 import React from "react";
 import Dropzone from "react-dropzone";
 
+const onFileDrop = (acceptedFile: any): void => {
+  const reader = new FileReader();
+  if (reader.error) {
+    alert(`Error occurred: ${reader.error}`);
+  }
+  reader.onload = () => {
+    try {
+      // eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+      //@ts-ignore
```

Comment:
why ?

Reply:
ts complains that the value of reader.result could be null

Reply:
Isn't this a valid complaint?